### PR TITLE
handle null argument list on invocation expression

### PR DESCRIPTION
**Pull request by jmarolf**
Fixes https://github.com/dotnet/roslyn/issues/13605

### CODE REVIEW

**Comment by jmarolf** [comment]
retest windows_eta_open_prtest please

**Comment by jmarolf** [comment]
test vsi please

**Comment by jmarolf** [comment]
@dotnet/roslyn-ide Review please

**Comment by Pilchie** [comment]
:+1:

File: src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs [hunk 1]
```diff
@@ -1693,7 +1693,8 @@ public bool IsStringLiteral(SyntaxToken token)
 
         public SeparatedSyntaxList<SyntaxNode> GetArgumentsForInvocationExpression(SyntaxNode invocationExpression)
         {
-            return ((invocationExpression as InvocationExpressionSyntax)?.ArgumentList.Arguments).Value;
+            var arguments = (invocationExpression as InvocationExpressionSyntax)?.ArgumentList?.Arguments;
+            return arguments.HasValue ? arguments.Value : default(SeparatedSyntaxList<ArgumentSyntax>);
```
Comment (CyrusNajmabadi) [inline src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs]: this can't happen in C#.

**Comment by CyrusNajmabadi** [comment]
VB side looks fine.  I would undo the C# side.

File: src/EditorFeatures/VisualBasicTest/CodeActions/ConvertToInterpolatedString/ConvertToInterpolatedStringTests.vb [hunk 2]
```diff
@@ -418,4 +418,16 @@ Module T
...
+
+    <WorkItem(13605, "https://github.com/dotnet/roslyn/issues/13605")>
+    <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertToInterpolatedString)>
+    Public Async Function TestInvocationWithNullArguments() As Task
+        Dim text = <File>
+Module Module1
+    Sub Main()
+        [|TaskAwaiter|]
+    End Sub
+End Module</File>.ConvertTestSourceTag()
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/VisualBasicTest/CodeActions/ConvertToInterpolatedString/ConvertToInterpolatedStringTests.vb]: note: you can just use a VB multiline string literal now.  That's the preferred style in Vb tests now.

Comment (DustinCampbell) [inline src/EditorFeatures/VisualBasicTest/CodeActions/ConvertToInterpolatedString/ConvertToInterpolatedStringTests.vb]: yes, please!

**Comment by CyrusNajmabadi** [comment]
:+1:

**Pull request merged**